docs/2018-ROADMAP.md

```diff
@@ -15,7 +15,7 @@ This document defines the roadmap for TiKV development.
 - [x] Raft Learner - Support Raft learner to smooth the configuration change process
 - [x] Raft Pre-vote - Support Raft pre-vote to avoid unnecessary leader election on network isolation
 - [ ] Joint Consensus - Change multi members safely
-- [ ] Multi-thread Raftstore - Process Region Raft logic in multiple threads
+- [X] Multi-thread Raftstore - Process Region Raft logic in multiple threads
 - [ ] Multi-thread Apply Pool - Apply Region Raft committed entries in multiple threads
```

Comment:
I think Multi-thread Apply Pool is done too.

Reply:
Indeed.

Reply:
Ok, will fill it